sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/jul/BaseJulLogger.java

```diff
@@ -71,7 +72,7 @@ protected Handler makeLoggingHandler(String logName, Logger heliumRecordLog) {
                     handler.setFormatter(formatter);
                     handler.setEncoding(logCharSet);
                 } catch (IOException e) {
-                    e.printStackTrace();
+                    RecordLog.warn("[BaseJulLogger]");
```

Comment:
It's not allowed to record logs in the following classes (to avoid potential circular class dependency):

- All util classes
- SentinelConfigLoader and relevant classes
- Log-related